tutorials/scripting/c_sharp/c_sharp_global_classes.rst

```diff
@@ -84,8 +91,7 @@ will let you create and load instances of this type easily.
 .. warning::
 
     The Godot editor will hide these custom classes with names that begin with the prefix
-    "Editor" in the 'Create New Node' or 'Create New Scene' dialog windows. The classes 
-    are available for instantiation at runtime via their class names, but are 
-    automatically hidden by the editor windows along with the built-in editor nodes used 
+    "Editor" in the 'Create New Node' or 'Create New Scene' dialog windows. The classes
+    are available for instantiation at runtime via their class names, but are
+    automatically hidden by the editor windows along with the built-in editor nodes used
```

Comment:
```suggestion
    "Editor" in the Create New Node or Create New Scene dialog windows. The classes
    are available for instantiation at runtime via their class names, but are
    automatically hidden by the editor windows along with the built-in editor nodes used
```
Use of single quotes `'` for the window names struck me as out of place. There's no real standard here, but no quotes seems most common.
![Code_IwuKlHLJl0](https://github.com/user-attachments/assets/3acb7d03-07d0-433d-8705-d92d212602c1)